packages/content-editor-plugin/src/components/Toolbar/InsertLink.tsx

```diff
@@ -118,35 +112,29 @@ export const InsertLinkDialog = () => {
 
   return (
     <Dialog onOpenChange={handleOpenChange} open={isInsertingLink}>
-      <form onSubmit={handleSubmit} noValidate>
-        <DialogTitle>Insert Link</DialogTitle>
-        <DialogContent>
-          <div className={styles.fullWidth}>
-            <FormField
-              label="Image URL text"
-              validationStatus={errors?.url ? 'error' : undefined}
-              helperText={errors?.url}
-            >
-              <Input value={values?.url} inputProps={{ name: 'url' }} onChange={handleChange} />
-            </FormField>
-            <FormField
-              label="Link text"
-              validationStatus={errors?.text ? 'error' : undefined}
-              helperText={errors?.text}
-            >
-              <Input value={values?.text} inputProps={{ name: 'text' }} onChange={handleChange} />
-            </FormField>
-          </div>
-        </DialogContent>
-        <DialogActions>
-          <ButtonBar>
-            <Button onClick={handleClose}>Cancel</Button>
-            <Button variant="cta" type="submit">
-              Insert
-            </Button>
-          </ButtonBar>
-        </DialogActions>
-      </form>
+      <DialogTitle>Insert Link</DialogTitle>
+      <DialogContent>
+        <div className={styles.fullWidth}>
+          <FormField validationStatus={errors?.url ? 'error' : undefined}>
+            <FormFieldLabel>Image URL text</FormFieldLabel>
+            <Input value={values?.url} inputProps={{ name: 'url' }} onChange={handleChange} />
+            <FormFieldHelperText>{errors?.url}</FormFieldHelperText>
+          </FormField>
+          <FormField validationStatus={errors?.text ? 'error' : undefined}>
+            <FormFieldLabel>Link Text</FormFieldLabel>
+            <Input value={values?.text} inputProps={{ name: 'text' }} onChange={handleChange} />
+            <FormFieldHelperText>{errors?.text}</FormFieldHelperText>
+          </FormField>
+        </div>
+      </DialogContent>
+      <DialogActions>
+        <ButtonBar>
```

Comment:
No need for ButtonBar in actions right now. Although the actions don't have the full functionality of the old button bar. But if this works well then maybe leave it. I haven't tried it myself.